CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ the code was deployed.
 
 ## [Unreleased]
 
+## [1.3.0] - 2020-12-22
```

Comment:
I'd also suggest just leaving this changelog message under the `[Unreleased]` section. The exact version number can be figured out when we do the prod deployment.